color/p5js/Color-Continuity/index.md

```diff
@@ -0,0 +1,9 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Law of Prägnanz, Law of Contrast, Law of Multistable-perception, Law of Preference
+tags: [P5.js, Law of Prägnanz, Law of Law of Similarity, Law of Common-Fate, Law of Connected-Elements, Law of Contrast, Law of Multistable-perception, Law of Relativity of Properties, Law of Similarity, Law of Simplicity, Law of Symmetry, Color, Chance, Interaction, Repeat]
+permalink: /Color/Continuity/Interaction/Multistable-perception/
```

Comment:
permalink lowercase. Should reflect the name of the sketch. No sub sub usb categories.